js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+function abbreviateString(str) {
+  if (typeof (str) !== 'string') {
+    throw Error(`Expected string, got ${typeof str}`);
+  }
+
+  if (str.trim().length < 1) {
+    throw Error('Got empty string.');
+  }
+
+  if (str.split(' ').length === 1) {
```

Comment:
For "Single   ", this going to be `false`.
You better need to change it to

```suggestion
  if (str.trim().split(' ').length === 1) {
```